java-frontend/src/test/java/org/sonar/java/model/JavaVersionImplTest.java

```diff
@@ -86,11 +91,24 @@ void java_12() {
     assertThat(version.isJava6Compatible()).isTrue();
     assertThat(version.isJava7Compatible()).isTrue();
     assertThat(version.isJava8Compatible()).isTrue();
+    assertThat(version.isJava9Compatible()).isTrue();
     assertThat(version.isJava12Compatible()).isTrue();
     assertThat(version.isJava15Compatible()).isFalse();
     assertThat(version.asInt()).isEqualTo(12);
   }
 
+  @Test
+  void java_15() {
+    JavaVersion version = new JavaVersionImpl(15);
+    assertThat(version.isJava6Compatible()).isTrue();
+    assertThat(version.isJava7Compatible()).isTrue();
+    assertThat(version.isJava8Compatible()).isTrue();
+    assertThat(version.isJava9Compatible()).isTrue();
+    assertThat(version.isJava12Compatible()).isTrue();
+    assertThat(version.isJava15Compatible()).isTrue();
+    assertThat(version.asInt()).isEqualTo(15);
+  }
```

Comment:
You added a test for Java 15, but not for Java 9, do you think it would be useful to add it?